web_widget_tolocalestring/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```

Comment:
sorry I did not understand

I took the model : https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst